torch/_inductor/fx_passes/post_grad.py

```diff
@@ -758,6 +769,20 @@ def can_inplace(node, mutated_arg):
             kwargs = dict(node.kwargs)
             kwargs["tensors_to_clone"] = tensors_to_clone
             node.kwargs = immutable_dict(kwargs)
+        elif (
```

Comment:
I think this might overlap with the logic here? https://github.com/pytorch/pytorch/pull/113215/files#diff-5d0c4891ce10ae399cd6d44f7408532d1a9eb56103ab0cd7db3613cdf4fbaf50R674